fix: add a default batch limit (#1150)
ava-labs/coreth: plugin/evm/config/config.md

Maximum size in bytes for HTTP request bodies. Defaults to `0` (no limit).

### `batch-request-limit`

_Integer_

> Maximum number of requests that can be batched in an RPC call. For no limit, set both this parameter and `batch-response-max-size` to 0. Defaults to `1000`.
> 
> ### `batch-response-max-size`
> 
> _Integer_
> 
> Maximum size (in bytes) of response that can be returned from a batched RPC call. For no limit, set both this parameter and `batch-request-limit` to 0. Defaults to `25 MB`.

Reviewer: ```suggestion
Maximum number of requests that can be batched in an RPC call. For no limit, set either this or `batch-response-max-size` to 0. Defaults to `1000`.

### `batch-response-max-size`

_Integer_

Maximum size (in bytes) of response that can be returned from a batched RPC call. For no limit, set either this or `batch-request-limit` to 0. Defaults to `25 MB`.
```

> Maximum size of response that can be returned from a batched RPC call. For no limit, set to 0. If set to 0, `batch-request-limit` must also be set to 0. Defaults to `25 MB`.

Reviewer: I feel like this doc isn't quite accurate

> `If set to 0, `batch-XXX` must also be set to 0.`

To me, that reads like the node will error if that isn't the case... In reality, we only enable the limits if both of them are non-0.

Author: I think this is more clear
